core/tests/test_utils.py

```diff
@@ -658,6 +658,16 @@ def count_jobs_in_taskqueue(self, queue_name=None):
         else:
             return len(self.taskqueue_stub.get_filtered_tasks())
 
+    def get_pending_tasks(self, queue_name=None):
```

Comment:
Maybe have count_jobs_in_taskqueue() refer to this method, in order to avoid duplication?